src/ProjectTemplates/test/MvcTemplateTest.cs

```diff
@@ -306,7 +306,6 @@ public async Task MvcTemplate_SingleFileExe()
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/23993")]
         public async Task MvcTemplate_RazorRuntimeCompilation_BuildsAndPublishes()
```

Comment:
I saw @Tratcher's comment at https://github.com/dotnet/aspnetcore/issues/23993#issuecomment-696427823. Do we need to take action on that?

Reply:
That test was already fixed in another PR https://github.com/dotnet/aspnetcore/pull/26825, the tracking for fixing tests is slightly awkward since they are filed in batches, but the fixes/issues aren't necessarily the same.

Reply:
Got it! Thanks for clarifying.